index.html

```diff
@@ -152,5 +152,16 @@ <h1 id="title">mcMerge</h1>
     <noscript>
       You need Javascript to proceed.
     </noscript>
+
+    <hr />
+
+    <div id="footer">
+      <div id="source-link">
+        <a href="https://github.com/mozilla/mcMerge"> mcMerge Source </a>
+      </div>
+      <div id="mcmerge-revision">
+        <a href="https://mcmerge.paas.allizom.org/deploy.txt"> Deployed revision </a>
```

Comment:
Would this be best as a relative link? Given the URL will be subject to change? I know the link will be broken locally, but I think that's perhaps more accurate locally anyway? Happy for you to make the final call on this :-)